gf-webapp/src/Index.scss

```diff
@@ -66,4 +66,53 @@ $input-padding: 10px;
 $input-font-size: 18px;
 $input-font-family: Montserrat Regular;
 $input-border: 2px solid $color-primary;
-$input-border-focus: 2px solid $color-secondary;
\ No newline at end of file
+$input-border-focus: 2px solid $color-secondary;
+
+//room titles styles
+$room-title-font-family: Montserrat Regular;
+
+//room h1 titles styles
+$room-h1-display: flex;
+$room-h1-flex-direction: column;
+$room-h1-align-items: center;
+$room-h1-justify-content: center;
+$room-h1-position: absolute;
+$room-h1-top: 0;
+$room-h1-left: 0;
+$room-h1-right: 0;
+$room-h1-bottom: 0;
```

Comment:
Try use 
inset: 0; instead of 4 line top, right, bottom, left

Reply:
Done